docs/user_guide.md

```diff
@@ -238,6 +240,38 @@ times and statistical results across these repetitions will also be reported.
 As well as the per-benchmark entries, a preamble in the report will include
 information about the machine on which the benchmarks are run.
 
+<a name="setup-teardown" />
+
+## Setup/Teardown
+
+Global setup/teardown specific to each benchmark group can be done by
```

Comment:
i think it's just "each benchmark".  ie, a benchmark function defines a benchmark family and the args/threads when enumerated define "benchmarks".